### ISSUE TITLE
[i18n]: Translators Needed

### ISSUE THREAD

**Issue by dlford** [developertools-tech/developertools.tech#67]
# Translations Needed

This issue is for tracking all translations needed, please feel free to comment if you'd like to see a language added to the list below, or contribute by adding translations from the list below.

All translation related pull requests will be counted for Hacktoberfest during the month of October.

## Spanish

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

## Chinese

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

## Japanese

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Lorem Ipsum (add a word list?)
- [ ] Markdown Preview tool
- [ ] RegEx Tester (test case input)
- [ ] Text Diff (diff options menu items)

## Hindi

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

## Arabic

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

## Portuguese

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

## Bengali

- [ ] Header & Footer (Dev Tools in title bar, sponsors, contributors)
- [ ] Navigation (tool titles in nav bar and on each tool page)
- [ ] Tools

**Comment by leslievelderrainx** [comment]
Hi, I can help with Spanish translations.

**Comment by dlford** [comment]
@leslievelderrainx thanks so much! There are instructions in the [Contributors' Guide](https://github.com/developertools-tech/developertools.tech/blob/main/CONTRIBUTING.md), but feel free to reach out if you have any questions!

**Comment by leslievelderrainx** [comment]
Hey, I'm really sorry but I don't think I have the skill set to help with this issue. I'm new to github and coding in general. Since this was a translation request, I thought there wouldn't be any coding involved, but I looked at the contributor's guide and saw that you need to be familiar with next.js and other things that I've never really heard of. I'm super confused in general. I tried looking up tutorials but they confused me even more. Again, I'm sorry but I don't know if I can help.

**Comment by dlford** [comment]
@leslievelderrainx you don't have to do all that, if you just want to copy the engilsh JSON files and change the words to Spanish that's all we really need, let me know if you still want to give it a shot, I'm happy to help if you need anything.

**Comment by leslievelderrainx** [comment]
Yea, I can definitely do that! Thanks for clarifying.

**Comment by jamessom** [comment]
Hi @dlford I can help with pPortuguese BR

**Comment by dlford** [comment]
@jamessom thanks so much!

**Comment by ayushsgit** [comment]
HI @dlford I can help with hindi

**Comment by LegendaryGene** [comment]
Hi @dlford I can help with Hindi as well!

**Comment by dlford** [comment]
Sorry @LegendaryGene, @ayushsgit was first

**Comment by ayushsgit** [comment]
@dlford thanks doing it :)

**Comment by Akash190104** [comment]
Hi @dlford assign me the bengali language. I can translate that.

**Comment by ayushsgit** [comment]
@dlford i translated en.json to Hindi, please add hacktoberfest label to it :)

**Comment by Wangata** [comment]
Hi @dlford I can help with Swahili translation

**Comment by fahadashiq12** [comment]
Hello, if you are looking for Urdu Language Support I am happy to help.

**Comment by dlford** [comment]
@Akash190104 @Wangata @fahadashiq12 please proceed, and thank you!

**Comment by kolja-lucht** [comment]
Hey @dlford, are you looking for German translation, too? I'd be happy to add those 🙂

**Comment by dlford** [comment]
@kolja-lucht yes please! Thanks!

**Comment by ultr4nerd** [comment]
Hey! Can I help with Spanish?

**Comment by dlford** [comment]
@ultr4nerd Absolutely! Thanks! I still need to add a few translation strings, can you wait until tomorrow to start? After that you can just check the `en` translation files for new fields.

**Comment by Ns-SheF** [comment]
Hiii....  I can translate to Hausa.  Currently mobile, will take a more comfortable look at the needed translations when on a system.  Assign this to me if you'd like to have Hausa also. Thanks.

**Comment by protob** [comment]
Hi, I can add Polish translation if applicable.

**Comment by dlford** [comment]
@Ns-SheF and @protob  yes, please feel free to proceed!

**Comment by ultr4nerd** [comment]
@dlford sure, I'll wait!

**Comment by lysnbrl** [comment]
@dlford, hi, can I help with the translation of common.json file?

**Comment by dlford** [comment]
@ultr4nerd yes, and also check all the `en` files for new fields if you don't mind please?

**Comment by dlford** [comment]
@lysnbrl what language?

**Comment by lysnbrl** [comment]
@dlford spanish!

**Comment by dlford** [comment]
@lysnbrl sorry, Spanish is assigned already.

**Comment by ultr4nerd** [comment]
@dlford we can split the work!

**Comment by dlford** [comment]
@ultr4nerd @lysnbrl please feel free to work out splitting the work if you both would like to do that

**Comment by protob** [comment]
@dlford thanks! I created pull request.

**Comment by arefathi** [comment]
Hello there. 
developer-tools looks like a very interesting project. I would like to make Amharic `am` translation contributions. 
Please let me know if that's okay. Thanks.

**Comment by giovannipessiva** [comment]
Hi @dlford, I can help with an Italian translation

**Comment by dlford** [comment]
@arefathi and @giovannipessiva please fell free to proceed, thank you!

**Comment by jennienguyendev** [comment]
Hi @dlford, I can help with Vietnamese translation if you decide to add it to the list.

**Comment by dlford** [comment]
@jennienguyendev yes please, all yours!

**Comment by harshjais369** [comment]
@dlford I can do for Indonesian translation, if you wish to add it to the list. Additionally, I would appreciate much if you could give me a brief guidance about where or which files I need to put my work on.

Thank you!..

**Comment by ivanignatiev** [comment]
@dlford I can work on Ukrainian translation

**Comment by dlford** [comment]
@ivanignatiev Thank you! All yours!

**Comment by kshjessica** [comment]
@dlford Hi! Can I work on Korean translation by any chance?

**Comment by goobric** [comment]
hi @dlford  can I also help with the German de-de translation?

**Comment by dlford** [comment]
@kshjessica all yours!

**Comment by dlford** [comment]
@goobric all yours!

**Comment by dlford** [comment]
@harshjais369 Yes! All yours!
